quiz/templates/quiz/quiz.html

```diff
@@ -0,0 +1,117 @@
+{% extends "_base.html" %}
+
+{% block content %}
+    <!-- general form elements -->
+    <div class="box box-primary">
+    <div class="box-header with-border">
+        <h3 class="box-title">Quiz</h3>
+    </div>
+    <!-- /.box-header -->
+    <!-- form start -->
+    <form role="form" onsubmit="return submitAnswer();">
+        {% csrf_token %}
+        <div class="box-body">
+        <div class="form-group">
+            <label id="question"></label>
+        </div>
+            <div id="answers"> </div> 
+        </div>
+        <!-- /.box-body -->
+        <div class="box-footer">
+            <button class="btn btn-primary">Submit</button>
+        </div>
+    </form>
+    </div>
+    <!-- /.box -->
+{% endblock content %}
+
+{% block javascript  %}
+    {{ block.super }}
+    <script>
+        $(document).ready(main());
+
+        function main(){
+            var url = "/quiz/json";
+
+            $.ajaxSetup({
+                beforeSend: function(xhr, settings) {
+                    if (!csrfSafeMethod(settings.type) && !this.crossDomain) {
+                        xhr.setRequestHeader("X-CSRFToken",  getCookie("csrftoken"));
+                    }
+                }
+            });
+
+            $.getJSON(
+                url,
+            )
+            .done(function(data){
+                setQuestion(data);
+            })
+        }
+
+        function submitAnswer(){
+            var selectedAnswerID = $("input[name='answer']:checked").val();
+            alert(selectedAnswerID);
```

Comment:
I like to use `console.log()` or `console.debug()` for debugging. This puts it into the javascript console.